src/PrestaShopBundle/Install/Install.php

```diff
@@ -874,16 +874,14 @@ public function configureShop(array $data = [])
         }
 
         // Update default contact
-        if (isset($data['admin_email'])) {
-            Configuration::updateGlobalValue('PS_SHOP_EMAIL', $data['admin_email']);
-            Configuration::updateGlobalValue('PS_LOGS_EMAIL_RECEIVERS', $data['admin_email']);
-
-            $contacts = new PrestaShopCollection('Contact');
-            /** @var \Contact $contact */
-            foreach ($contacts as $contact) {
-                $contact->email = $data['admin_email'];
-                $contact->update();
-            }
+        Configuration::updateGlobalValue('PS_SHOP_EMAIL', $data['admin_email']);
```

Comment:
where is this condition now ?

Reply:
Check the l.849.... If `!isset($data['admin_email']`, then go l.871